vividus-plugin-lighthouse/src/test/java/org/vividus/lighthouse/LighthouseStepsTests.java

```diff
@@ -276,6 +282,11 @@ private PagespeedApiPagespeedResponseV5 mockResponse(ArrayMap<String, BigDecimal
         when(lighthouseResultV5.getAudits()).thenReturn(Map.of("metrics", lighthouseAuditResultV5));
         List<ArrayMap<String, BigDecimal>> items = List.of(metrics);
         when(lighthouseAuditResultV5.getDetails()).thenReturn(Map.of("items", items));
+        Categories categories = mock();
+        when(lighthouseResultV5.getCategories()).thenReturn(categories);
+        LighthouseCategoryV5 category = mock();
+        when(categories.getPerformance()).thenReturn(category);
+        when(category.getScore()).thenReturn(new BigDecimal(99));
```

Comment:
where is it asserted/verified?

Reply:
https://github.com/vividus-framework/vividus/pull/4095/files#diff-96e79817ad71c32edb9217833529f8758e0b5bc7ac3d85f0c1f9ce3107c9fa78R128

Reply:
where is a check of number like `99`?

Reply:
expected metric value is passed to validateMetric method and validated there